magento/adobe-stock-integration
# Improve media gallery styles && directory filters state

### Fixed Issues (if relevant)
1. magento/adobe-stock-integration#1217: Move folder name lower in the folder tree
2. magento/adobe-stock-integration#1220: Add distance between image name, ST symbol, extension. [All web browsers]
3. magento/adobe-stock-integration#1222 "Active filters: Directory" remains applied when the last folder in Media Gallery has been deleted
4. magento/adobe-stock-integration#1221 Directory filter consider image names 
5. magento/adobe-stock-integration#1232 Image description doesn't match the UI Prototype 
6. magento/adobe-stock-integration#1205: Search by keywords does not work
7. magento/adobe-stock-integration#1187: The image from Media Gallery is not found via "Search by keyword" if a part of name is searched for

### Manual testing scenarios (*)
1. From Admin go to **Content** - **Media - Media Gallery** 
2. Observe the images names and extensions'

The folder name is closer to the center of the row
Distance between image name, ST symbol, extension
